dev/make-distribution.sh

```diff
@@ -156,7 +156,7 @@ BUILD_COMMAND=("$MVN" -T 1C clean package -DskipTests $@)
 echo -e "\nBuilding with..."
 echo -e "\$ ${BUILD_COMMAND[@]}\n"
 
-"${BUILD_COMMAND[@]}"
+# "${BUILD_COMMAND[@]}"
```

Comment:
whoops.  I do this during development so I can get a distribution AND use incremental builds.  removed.